resources/ports.yml

```
@@ -1579,6 +1581,13 @@ ports:
     color: sky
     current-maintainers: []
     past-maintainers: [*isabelroses]
+  sent:
+    name: sent
+    categories: [document_viewer, productivity]
+    platform: [linux]
+    color: text
```

```suggestion
    color: blue
```
Oops meant to change this with the icon addition.